vanity/content/code/mixin/cowsay.md

```diff
@@ -1,5 +1,5 @@
 ---
 title: "cowsay mixin"
-vanity: "https://github.com/deislabs/porter-cowsay"
+vanity: "https://github.com/carolynvs/porter-cowsay"
```

Comment:
I think cowsay should be removed from the vanity server. 🤔 It's not an official mixin anymore. Maybe something to think about later

Reply:
```
 ______________
< Estoy triste >
 --------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||
```